ftw/mobilenavigation/browser/resources/navigation.css

```diff
@@ -139,6 +139,15 @@ div.slideNavi.loading {
   min-height: 200px;
   display: block;
 }
+#slider-container {
+  overflow: auto;
+  position: fixed;
+  top:      56px;
+  height:   100%;
+  height:   calc(100% - 56px);
```

Comment:
You probably mean padding-top: 56px
BUT even so - this doesn't work, because the padding hides toggling menu button from mouse clicks.
Any other suggestions?